### ISSUE TITLE
[Design Update] Material Design 3

### ISSUE THREAD

**Issue by kurokawaerne** [foobnix/LibreraReader#1216]
### Are there any plans on updating this app to modern android design standards? 

**Material Design 3** (MD3) is there since **2021**, with the release of Android 12. This new standard gives an uniform look between apps and the system. 

You should definitely take a look at it on m3.material.io , Google says it's easy to migrate, so why not give your app its well deserved design refresh?

**Comment by d3rpd3rpd3rp** [comment]
I agree, the Librera UI could definitely benefit from an upgrade. Although I always prefer functionality over appearance, the app deserves a new polish and the current interface is a little austere.

**Comment by viasux** [comment]
I'd be willing to help with prototyping and wireframing if requested.

**Comment by kamik-hub** [comment]
This app really needs UI remake for latest Android style like Material 3 Expressive
